Gemfile

```diff
@@ -237,7 +237,7 @@ group :development, :test do
   gem 'byebug'
   gem 'pry-rails'
   gem 'bundler-audit', require: false
-  gem 'brakeman', '~> 5.0.4', require: false
+  gem 'brakeman', require: false
```

Comment:
Thanks!  I wonder why we'd pinned it.  Probably intended to >= 5.0.4...  This works great though.